assets/js/components/Dashboard/PermanentConfiguration/Workflows/GlEink/GlEinkWorkflow.tsx

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React, { ComponentType, useState } from "react";
+import { WorkflowProps } from "../../ConfigureScreensPage";
+import ConfigureScreensWorkflowPage, {
+  PlaceIdsAndScreens,
+} from "./ConfigureScreensPage";
+import BottomActionBar from "../../BottomActionBar";
+import { useNavigate } from "react-router-dom";
+import StationSelectPage from "./StationSelectPage";
+import { Place } from "../../../../../models/place";
+
+const GlEinkWorkflow: ComponentType<WorkflowProps> = ({
+  places,
+}: WorkflowProps) => {
+  const [selectedPlaces, setSelectedPlaces] = useState<Set<Place>>(new Set());
+  const [configVersion, setConfigVersion] = useState<string>("");
+
+  const [placesAndScreensToUpdate, setPlacesAndScreensToUpdate] =
+    useState<PlaceIdsAndScreens>({});
+
+  const navigate = useNavigate();
+  const [configStep, setConfigStep] = useState<number>(0);
+
+  const handleRemoveLocation = (place: Place) => {
+    const newSelectedPlaces = new Set(selectedPlaces);
+    newSelectedPlaces.delete(place);
+    setSelectedPlaces(newSelectedPlaces);
+    setPlacesAndScreensToUpdate((placesAndScreens) => {
+      delete placesAndScreens[place.id];
+      return placesAndScreens;
+    });
+  };
+
+  let backButtonLabel;
+  let forwardButtonLabel;
+  let cancelButtonLabel;
+  let onBack;
+  let onForward;
+  let onCancel;
+  let layout;
+  const forwardButtonDisabled = selectedPlaces.size === 0;
+  switch (configStep) {
+    case 0:
+      cancelButtonLabel = "Cancel";
+      forwardButtonLabel = "Next";
+      onCancel = () => {
+        navigate(-1);
+      };
+      onForward = () => {
+        setConfigStep(configStep + 1);
+      };
+      layout = (
+        <StationSelectPage
+          places={places}
+          selectedPlaces={selectedPlaces}
+          setSelectedPlaces={setSelectedPlaces}
+        />
+      );
+      break;
+    case 1:
+      backButtonLabel = "Back";
+      forwardButtonLabel = "Review Screens";
+      cancelButtonLabel = "Cancel";
+      onCancel = () => {
+        navigate(-1);
+      };
+      onBack = () => {
+        setConfigStep(configStep - 1);
+      };
+      onForward = () => {
+        navigate("/pending");
+      };
+      layout = (
+        <ConfigureScreensWorkflowPage
+          selectedPlaces={Array.from(selectedPlaces)}
```

Comment:
Mostly because it no longer needs to be a set after this page. Working with the set as an array makes mapping over the elements a little more familiar.